pages/[lang]/index.js

```diff
@@ -0,0 +1,281 @@
+import Container from '../../components/layout/Container'
+import NavBar from '../../components/navigation/NavBar'
+import Hero from '../../components/Hero'
+import NewsletterSubscribe from '../../components/NewsletterSubscribe'
+import Sponsors from '../../components/sponsors/Sponsors'
+import Head from '../../components/Head'
+import Slack from '../../components/slack'
+import Button from '../../components/buttons/Button'
+import Calendar from '../../components/Calendar'
+import AdidasLogo from '../../components/logos/Adidas'
+import AxwayLogo from '../../components/logos/Axway'
+import SlackLogo from '../../components/logos/Slack'
+import SalesforceLogo from '../../components/logos/Salesforce'
+import SapLogo from '../../components/logos/SAP'
+import Testimonial from '../../components/Testimonial'
+import Heading from '../../components/typography/Heading'
+import Paragraph from '../../components/typography/Paragraph'
+import TextLink from '../../components/typography/TextLink'
+import GoldSponsors from '../../components/sponsors/GoldSponsors'
+import SilverSponsors from '../../components/sponsors/SilverSponsors'
+import SupportUs from '../../components/SupportUs/SupportUs'
+import StickyNavbar from '../../components/navigation/StickyNavbar'
+import GoogleCalendarButton from '../../components/buttons/GoogleCalendarButton';
+import ICSFileButton from '../../components/buttons/ICSFileButton';
+import SubscribeButton from '../../components/buttons/SubscribeButton';
+import NewsroomSection from '../../components/newsroom/NewsroomSection'
+import {
+  getAllLanguageSlugs,
+  getLanguage,
+} from "next-i18next-static-site";
+
+function HomePage() {
+  return (
+    <>
+      <Head />
+      <StickyNavbar>
+        <NavBar className="max-w-screen-xl block px-4 sm:px-6 lg:px-8 mx-auto" />
+      </StickyNavbar>
+
+      <main id="main-content" className="scroll-mt-5">
+        <Container wide>
+          <Hero className="mb-24" />
+        </Container>
+
+        <Container className="text-center pb-12" wide as="section">
+          <Heading
+            level="h3"
+            typeStyle="heading-lg"
+            className="mb-4"
+          >
+            Adopted by the world leading brands
+          </Heading>
+          <Paragraph className="mt-2 mb-20 md:w-2/3 md:mx-auto">
+            These brands are already using AsyncAPI in production. Is your company using AsyncAPI and wants to be included in this list?
```

Comment:
Aren't we targeting to add the language variables for the webpages in this PR?

Reply:
No. The aim of this PR is to initialize the translation hooks and check if everything's working.
The translation would be done in subsequent PRs.